### PR TITLE
Add a failing test case for a VCF with no format or genotypes

### PR DESCRIPTION
**Pull request by cmdcolin**
I was working with Rob on breakend concepts and found this repo had some nice test data

Here is an example file that was failing parsing, and a utility function to just parse a simple VCF file for tests

https://github.com/PapenfussLab/StructuralVariantAnnotation/tree/master/inst/extdata

### CODE REVIEW

File: src/parse.js [hunk 1]
```diff
@@ -220,7 +222,10 @@ class VCF {
     } else {
       variant.FILTER = fields[6].split(';')
     }
-    const info = fields[7] === '.' ? {} : this._parseKeyValue(fields[7])
+    const info =
+      fields[7] === undefined || fields[7] === '.'
```
[the remark this answers is not in the document]
Comment (cmdcolin) [reply src/parse.js]: I'll just "throw" it out there: is it worth manually throwing an exception in this case if we can reasonably gather enough information to partiallly parse the file just to be fully spec compliant?

File: src/parse.js [hunk 2]
```diff
@@ -4,11 +4,19 @@ import vcfReserved from './vcfReserved'
  * Class representing a VCF parser, instantiated with the VCF header.
  * @param {object} args
  * @param {string} args.header - The VCF header. Supports both LF and CRLF
+ * @param {boolean} args.strict - Whether to parse in strict mode or not
```
Comment (garrettjstevens) [inline src/parse.js]: This needs to move down a line so "newlines" from the line below goes with the `header` option.

File: test/data/multipleAltSVs.vcf [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+##INFO=<ID=EVENT,Number=1,Type=String,Description="ID of event associated to breakend">
+##INFO=<ID=HOMLEN,Number=.,Type=Integer,Description="Length of base pair identical micro-homology at event breakpoints">
+##INFO=<ID=HOMSEQ,Number=.,Type=String,Description="Sequence of base pair identical micro-homology at event breakpoints">
+##INFO=<ID=IMPRECISE,Number=0,Type=Flag,Description="Imprecise structural variation">
+##INFO=<ID=MATEID,Number=.,Type=String,Description="ID of mate breakends">
+##INFO=<ID=SVLEN,Number=.,Type=Integer,Description="Difference in length between REF and ALT alleles">
+##INFO=<ID=SVTYPE,Number=1,Type=String,Description="Type of structural variant">
+##contig=<ID=chr1,length=249250621>
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
+chr1	1	.	A	<DEL>	.	.
```
[the remark this answers is not in the document]
Comment (cmdcolin) [reply test/data/multipleAltSVs.vcf]: I am now making it throw if strict mode is on (default on)

**Comment by cmdcolin** [comment]
I think this is ready now, it has the "strict" modifier to the parser now

**Pull request merged**